### PR TITLE
feat: add mail contact

### PR DESCRIPTION
**Pull request by tilmx**

### CODE REVIEW

**Comment by marionebl** [comment]
Deploy preview processing.

Built with commit d36b306392369f3a77ec4ef03b6add3aa7c441ac

https://app.netlify.com/sites/determined-cray-3fd905/deploys/5a3e96d7df99534d32afb56b

File: lib/index.html [hunk 1]
```diff
@@ -154,7 +154,8 @@ <h3>Prototype design with live data</h3>
 			<h2>Be part of shaping the future of design</h2>
 			<p>Help us taking Alva to the next level with your design expertise, product management insights or engineering skills. Our next features, bugs and the roadmap are fully open. Feel free to add issues, start pull requests or let us discuss what should be next.</p>
 
-			<a class="button" href="https://github.com/meetalva/alva/projects/3" target="_blank">Explore our next steps on Github</a>
+			<a class="button" href="https://github.com/meetalva/alva/projects/3" target="_blank">Explore our next steps on Github</a><br>
+			<a class="contribute-mail" href="mailto:hey@meetalva.io">or send us an Email</a>
```
Comment (marionebl) [inline lib/index.html]: A title attribute would be nice

File: lib/index.html [hunk 2]
```diff
@@ -206,6 +207,7 @@ <h4>Tilman Frick</h4>
 	</section>
 
 	<section class="section footer">
+		<a href="mailto:hey@meetalva.io">hey@meetalva.io</a><br><br>
```
Comment (marionebl) [inline lib/index.html]: A title attribute would be nice

**Comment by tilmx** [comment]
@marionebl I just added a title tag to the mail links

**Review by marionebl — APPROVED** [review]

**Pull request merged**